nautobot/docs/installation/upgrading-from-nautobot-v1.md

```diff
@@ -201,6 +217,7 @@ Below is a table documenting [removed redundant filter field changes](../release
 
 | Model              | Removed Filter Field | UI and API endpoints that are no longer supported in v2.X                             |
 |--------------------|----------------------|---------------------------------------------------------------------------------------|
+| Circuit            | `type_id`            | instead of `circuits/circuits/?type_id=<uuid>`, use `circuit_type=<uuid>`             |
```

Comment:
```suggestion
| Circuit            | `type_id`            | instead of `/circuits/circuits/?type_id=<uuid>`, use `circuit_type=<uuid>`            |
```